src/Auth0.OidcClient.Core/Auth0.OidcClient.Core.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
```

Comment:
Why does this need net6.0, when it's actually netstandard?

Reply:
It is true, it is not necessary